code/datums/quirks/negative_quirks/negative_quirks.dm

```diff
@@ -445,6 +447,74 @@
 		quirk_holder.toggle_move_intent()
 	quirk_holder.add_mood_event("nyctophobia", /datum/mood_event/nyctophobia)
 
+/datum/quirk/photophobia
+	name = "Photophobia"
+	desc = "Bright lights irritate you. Your eyes start to water and burn when exposed to light. Maybe it's a medical condition."
```

Comment:
The first two sentences seem like they're basically saying the same thing, can we cut one or merge them